libs/ledger-live-common/src/generated/types.ts

```diff
@@ -1,76 +1,152 @@
 import { TransactionRaw as polkadotTransactionRaw } from "../families/polkadot/types";
 import { TransactionStatus as polkadotTransactionStatus } from "../families/polkadot/types";
 import { TransactionStatusRaw as polkadotTransactionStatusRaw } from "../families/polkadot/types";
+import { RippleAccount } from "../families/ripple/types";
+import { RippleAccountRaw } from "../families/ripple/types";
 import { Transaction as rippleTransaction } from "../families/ripple/types";
 import { TransactionRaw as rippleTransactionRaw } from "../families/ripple/types";
 import { TransactionStatus as rippleTransactionStatus } from "../families/ripple/types";
 import { TransactionStatusRaw as rippleTransactionStatusRaw } from "../families/ripple/types";
+import { SolanaAccount } from "../families/solana/types";
+import { SolanaAccountRaw } from "../families/solana/types";
 import { Transaction as solanaTransaction } from "../families/solana/types";
 import { TransactionRaw as solanaTransactionRaw } from "../families/solana/types";
 import { TransactionStatus as solanaTransactionStatus } from "../families/solana/types";
 import { TransactionStatusRaw as solanaTransactionStatusRaw } from "../families/solana/types";
+import { StellarAccount } from "../families/stellar/types";
+import { StellarAccountRaw } from "../families/stellar/types";
 import { Transaction as stellarTransaction } from "../families/stellar/types";
 import { TransactionRaw as stellarTransactionRaw } from "../families/stellar/types";
 import { TransactionStatus as stellarTransactionStatus } from "../families/stellar/types";
 import { TransactionStatusRaw as stellarTransactionStatusRaw } from "../families/stellar/types";
+import { TezosAccount } from "../families/tezos/types";
+import { TezosAccountRaw } from "../families/tezos/types";
 import { Transaction as tezosTransaction } from "../families/tezos/types";
 import { TransactionRaw as tezosTransactionRaw } from "../families/tezos/types";
 import { TransactionStatus as tezosTransactionStatus } from "../families/tezos/types";
 import { TransactionStatusRaw as tezosTransactionStatusRaw } from "../families/tezos/types";
+import { TronAccount } from "../families/tron/types";
+import { TronAccountRaw } from "../families/tron/types";
 import { Transaction as tronTransaction } from "../families/tron/types";
 import { TransactionRaw as tronTransactionRaw } from "../families/tron/types";
 import { TransactionStatus as tronTransactionStatus } from "../families/tron/types";
 import { TransactionStatusRaw as tronTransactionStatusRaw } from "../families/tron/types";
 
+export type CoinAccount =
+  | AlgorandAccount
+  | BitcoinAccount
+  | CardanoAccount
+  | CeloAccount
+  | CosmosAccount
+  | CryptoOrgAccount
+  | ElrondAccount
+  | EthereumAccount
+  | FilecoinAccount
+  | HederaAccount
+  | NeoAccount
+  | OsmosisAccount
+  | PolkadotAccount
+  | RippleAccount
+  | SolanaAccount
+  | StellarAccount
+  | TezosAccount
+  | TronAccount;
```

Comment:
i'm not sure about this idea. I think we want to move out of this model where we do generated union types. instead the parametric type could live with the bridge (as a parametric type)